<launch>

    <include
        file="$(find hls_lfcd_lds_driver)/launch/hlds_laser.launch"
        clear_params="false"
        pass_all_args="true"/>

    <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
        <param name="port" value="/dev/ttyUSB1"/>
        <param name="baud" value="115200"/>
    </node>

    <node pkg="koresh" type="joy_node" name="joy_node" />
    <node pkg="koresh" type="robot_node" name="robot_node" />

    <node pkg="tf" type="static_transform_publisher" name="base_link_laser" args="0.15 0 0.14 0 3.1415926 3.1415926 base_link laser 20" />

</launch>